techniques/failures/F2.html

```diff
@@ -31,7 +31,7 @@ <h3>Images of text used as headings where the images are not marked up with head
             <p>Chapter1.gif is an image of the words, "Chapter One" in a Garamond font sized at 20 pixels. This is a failure because at a minimum the img element should be enclosed within a header element. A better solution would be to eliminate the image and to enclose the text within a header element which has been styled using CSS.</p>
          
 <pre xml:space="preserve"><code class="language-html">&lt;img src="Chapter1.gif" alt="Chapter One"&gt;
-&lt;p&gt;Once upon a time in the land of the Web...&lt;/p&gt;</code></pre>
+&lt;p&gt;Once upon a time in the land of the web...&lt;/p&gt;</code></pre>
```

Comment:
This feels like it verges on being more like proper noun usage? 
> lowercase except if proper noun meaning World Wide Web

Reply:
Especially since context is as title for a chapter.